test/common_utils.py

```diff
@@ -411,6 +411,21 @@ def create_cancer_data():
     return x_train, x_test, y_train, y_validation, feature_names, target_names
 
 
+def create_cancer_data_booleans():
+    # Import cancer dataset
+    cancer = retrieve_dataset('breast-cancer.train.csv', na_values='?').interpolate().astype('int64')
+    cancer_target = cancer.iloc[:, 0]
+    cancer_data = cancer.iloc[:, 1:]
+    feature_names = cancer_data.columns.values
+    target_names = [False, True]
+    cancer_target_list = cancer_target.tolist()
+    cancer_target_list = [False if label == 0 else True for label in cancer_target_list]
```

Comment:
Nit: since the classes are `0` and `1` you could simplify this with `cancer_target = cancer_target.astype(bool)` and also avoid changing the type from array to list.

Reply:
great idea, done!